### ISSUE TITLE
Broken rendering of 'usage' page on readthedocs

### ISSUE THREAD

**Issue by EFord36** [readthedocs/sphinx-hoverxref#280]
Hi,

I was reading the docs at https://sphinx-hoverxref.readthedocs.io/en/latest/usage.html

and noticed the page "Tooltip with content that needs extra rendering steps" onwards rendering strangely:

![image](https://github.com/readthedocs/sphinx-hoverxref/assets/20516159/96eada1c-caab-40a5-9166-5298e1e4d99a)

that screenshot was for Chrome on an M2 Mac, it also affects safari in the same way:

![Screenshot 2024-02-09 at 11 33 41](https://github.com/readthedocs/sphinx-hoverxref/assets/20516159/3e460a52-9344-4b3c-8b97-6994b5405e46)

This seems unintentional to me, so thought I would open an issue.

I had a quick look but didn't see an obvious problem - is it perhaps the use of the `.. bibliography::` directive in the previous section causing the issue, perhaps [sphinxcontrib-bibtex](https://github.com/mcmtroffaes/sphinxcontrib-bibtex) expects the bibliography to always be at the end of a page? I had a quick look at that code and couldn't see a smoking gun either, but I could just not know what I'm looking at.

**Comment by humitos** [comment]
Thanks for reporting this. Would you like to take a look at this and propose a PR to fix it? 😄

**Comment by EFord36** [comment]
Sure, I can give it a go!

**Comment by EFord36** [comment]
I've just looked at this, and actually, installing fresh and building, there's no problems. I diff-ed the live docs with my built version, and saw that the live docs were built with docutils 0.18.0. Downgrading to that version and building locally causes the error, so it looks like it's the docutils version (combined with other versions of packages presumably) that's the problem. I opened #281 to add a minimum version of docutils, but presumably anything that triggers a rebuild of the live readthedocs docs would fix the current problem (assuming it ends up with a fresh enough python environment).
